libs/components/code-examples/src/lib/modules/action-bars/summary-action-bar/error/example.component.ts

```diff
@@ -0,0 +1,108 @@
+import { Component, computed, inject } from '@angular/core';
+import { takeUntilDestroyed, toSignal } from '@angular/core/rxjs-interop';
+import { FormBuilder, FormControl, ReactiveFormsModule } from '@angular/forms';
+import {
+  SkySummaryActionBarError,
+  SkySummaryActionBarModule,
+} from '@skyux/action-bars';
+import { SkyKeyInfoModule } from '@skyux/indicators';
+
+interface donationSummary {
+  value: number;
+  label: string;
+}
+
+/**
+ * @title Summary action bar with errors
+ * @docsDemoHidden
+ */
+@Component({
+  selector: 'app-action-bars-summary-action-bar-error-example',
+  templateUrl: './example.component.html',
+  imports: [SkyKeyInfoModule, SkySummaryActionBarModule, ReactiveFormsModule],
+})
+export class ActionBarsSummaryActionBarErrorExampleComponent {
+  public donationSummary = new FormControl<donationSummary[] | null>([
+    {
+      value: 250,
+      label: 'Given this month',
+    },
+    {
+      value: 1000,
+      label: 'Given this year',
+    },
+    {
+      value: 1300,
+      label: 'Given all time',
+    },
+  ]);
+  public exampleForm = inject(FormBuilder).group({
+    donationSummary: this.donationSummary,
+  });
+
+  public valueChanged = toSignal(
```

Comment:
Is this being used?

Reply:
yeah that's what's having the computed value to update 😭